samples/simulated_device_cloud_upload/README.md

```diff
@@ -1,149 +1,6 @@
-# Simulated Device Cloud Upload sample for Azure IoT Edge
+# Azure IoT Edge - Simulated device-to-cloud sample
 
-These are step by step instructions on how to build and run this sample. A more in depth walkthrough, for 
-both [Linux](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-linux-gateway-sdk-simulated-device) and
-[Windows](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-windows-gateway-sdk-simulated-device), with 
-concept explanations and code snippets can be found on [docs.microsoft.com](https://docs.microsoft.com/en-us/azure/iot-hub/).
-
-This sample contains the following modules:
-
-1. A logger module
-2. Two SIMULATED_DEVICE module instances 
-3. An identity mapping module
-4. An IoT Hub module
-
-How it works: the two simulated device modules will produce messages containing simulated temperature readings and MAC address.
-These messages are received by the identity mapping module, which will add the device ID and key to the messages. Once messages
-carrying device ID, key, and temperature are received by the IotHub module, they will be sent to IoT Hub.
-
-The logger module logs all the exchanges on the message broker to a file. 
-
-## How to build the sample:
-### Linux
-
-1. Open a shell
-2. Navigate to `iot-edge/tools/`
-3. Run `build.sh`
-
->Note: `build.sh` does multiple things. It builds the project and places it in the "build" folder in the root of the repo. This folder is deleted and recreated every time `build.sh` is run. Additionally, `build.sh` runs all tests. The project can be build manually by using cmake. To do this:
-
->create a folder for the build output and navigate to that folder.
-
->run `sudo cmake <path to repo root>`
-
->Note: The cmake scripts will check for installed cmake dependencies on your machine before it attempts to fetch, build, and install them for you.
-
->run `make -j $(nproc)`
-
->To run the tests:
-
->run `ctest -j $(nproc) -C Debug --output-on-failure`
-
-### Windows
-
-1. Open a Developer Command for VS2015 as an Administrator
-2. Navigate to `iot-edge\tools\`
-3. Run `build.cmd`
-
->Note: `build.cmd` does multiple things. It builds a solution ('azure_iot_gateway_sdk.sln') and places it in the "build" folder in the root of the repo. This folder is deleted and recreated every time `build.cmd` is run. Additionally, `build.cmd` runs all tests. The project can be build manually by using cmake. To do this:
-
->create a folder for the build output and navigate to that folder.
-
->run `cmake <path to repo root>`
-
->Note: The cmake scripts will check for installed cmake dependencies on your machine before it attempts to fetch, build, and install them for you.
-
->run `msbuild /m /p:Configuration="Debug" /p:Platform="Win32" azure_iot_gateway_sdk.sln`
-
->To run the tests:
-
->run `ctest -C Debug -V`
-
-### Yocto Linux and Intel Edison
```

Comment:
I don't thing the walk-throughs on Microsoft.com cover Yocto and Edison, do they?